backend/core/api_serializers/relation_api_serializer.py

```diff
@@ -0,0 +1,72 @@
+# Third-party libraries
+from rest_framework import serializers
+
+# Local libraries
+from ..models import *
+from .author_api_serializer import AuthorAPISerializer
+from ..serializers.authorserializer import AuthorSerializer
+from ..serializers.relationserializer import RelationSerializer
+
+# Model Serializers 
+author_serializer = AuthorSerializer()
+relation_serializer = RelationSerializer()
+
+# API serializers
+author_api_serializer = AuthorAPISerializer()
+
+
+class RelationAPISerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Relation
+        fields = "__all__"
+    
+    def get_all_followers(self, author_id):
+        """
+        Returns a dictionary containing all the follower of author with author_id
```

Comment:
We should also check the other way around, like if a person1: sends a friend request to person2, then person1 would be from_author and person2 would be to_author and if person2 send back a request then we currently not storing a new instance.

Reply:
If person1 follows person2 and we need to get followers of person2 then it would only be person1